requirements.txt

```diff
@@ -220,7 +220,7 @@ mutagen==1.45.1
     # via -r requirements.in
 mypy-extensions==0.4.3
     # via typing-inspect
-numpy==1.21.5
```

Comment:
we changed the NumPy version because Blake's python version was different from the version on the text file and it prevented us from running the program. However we realized that for schemas we don't really need to run the program (if I understand correctly), and I believe we just simply forgot to change the NumPy version back to what it originally was!